### PR TITLE
Dark mode

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Added dark mode css variables and switch to setup page.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

**Comment by shjortConcordium** [comment]
It seems that the dark mode is only enabled after opening the setup page.
(So sign requests are always online)

**Review by shjortConcordium — CHANGES_REQUESTED** [review]
I assume that this is just for setting up the dark-mode system, seeing as the input fields and buttons don't change.
But please fix the fact that dark mode only starts when entering the setup page! :upside_down_face:

**Review by shjortConcordium — APPROVED** [review]

File: packages/browser-wallet/src/popup/shell/Root.tsx [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+    useEffect(() => {
+        if (theme === ThemeType.Light) {
+            document.getElementsByTagName('body').item(0)?.classList.remove('dark');
+        } else {
+            document.getElementsByTagName('body').item(0)?.classList.add('dark');
+        }
+    }, [theme]);
+
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    return <>{children}</>;
```
[the remark this answers is not in the document]
Comment (soerenbf) [reply packages/browser-wallet/src/popup/shell/Root.tsx:24]: Can't do that as we need to return JSX and not a react node.

[the remark this answers is not in the document]
Comment (shjortConcordium) [reply packages/browser-wallet/src/popup/shell/Root.tsx:24]: Hmm :thinking: 
It worked for me.

**Pull request merged**